app/scripts/modules/ui/DataView.js

```diff
@@ -181,18 +196,22 @@ DataView.prototype._generateHTMLForKeyValuePair = function (key, currentView) {
         };
     }
 
-    if (value && typeof value === 'object') {
-        valueHTML = JSONFormatter.formatJSONtoHTML(value);
+    if (oValue && oValue === 'object') {
```

Comment:
What is `oValue` variable holding - `string` or `object` ? 
Maybe it should be named `sValue` or the `typeof` is still needed.